### PR TITLE
Js workbench

### PR DESCRIPTION
**Pull request by tstangenberg**
Generated projects use the js workbench to refresh/update the browsers js files every time your Scala.Js project compiles.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,6 +27,19 @@ In case of the frontend-only project, you can use the sbt compile command to com
...
-While developing, you can use the *~compile* task, which will automatically recompile changed source files.
-
 Read more in the [Udash Developer's Guide](http://guide.udash.io/#/bootstrapping/generators).
+
+### ScalaJS Workbench
+The [ScalaJS Workbench](https://github.com/lihaoyi/workbench) is enabled by default.
+While developing the frontend code will be recompiled and updated in the browser automatically when source files changes are saved.
+
+#### Frontend only project
+1. Execute `sbt ~compileStatics`.
```
Comment (Starzu) [inline README.md]: I think you can simply use `~compile`.

**Comment by Starzu** [comment]
@tstangenberg It looks good, thanks! There are two minor things to be done before merge:
- rebase and squash your commits, please
- I think that workbench should be optional => there should be additional decision in project configuration (I can add it if you don't have time to work on this - just let me know)

**Comment by tstangenberg** [comment]
@Starzu Thanks for the feedback. I will implement it as optional.

**Comment by tstangenberg** [comment]
@Starzu The Workbench integration is now optional.

**Comment by Starzu** [comment]
@tstangenberg LGTM, thank you again!

**Pull request merged**